web/tests/libtest/env.py

```diff
@@ -257,15 +256,19 @@ def parts_to_url(codechecker_cfg):
 
 
 def get_workspace(test_id='test'):
-    tmp_dir = os.path.join(REPO_ROOT, 'build')
-    base_dir = os.path.join(tmp_dir, 'workspace')
-    if not os.path.exists(base_dir):
-        os.makedirs(base_dir)
+    """ return a temporary workspace for the tests """
+    workspace_root = os.environ.get("CC_TEST_WORKSPACE_ROOT")
```

Comment:
It would be great to mention this environment variable in the documentation (https://github.com/Ericsson/codechecker/blob/master/docs/tests/README.md)